parsers/cxx/cxx_parser_internal.h

```diff
@@ -220,7 +220,9 @@ typedef enum _CXXParserKeywordState
 	// "const" has been seen at block level
 	CXXParserKeywordStateSeenConst = (1 << 9),
 	// "volatile" has been seen at block level
-	CXXParserKeywordStateSeenVolatile = (1 << 10)
+	CXXParserKeywordStateSeenVolatile = (1 << 10),
+	// __attribute__((deprecated)) has been seen
+	CXXParserKeywordStateSeenAttributeDeprecated = (1 << 11)
```

Comment:
I would like to use C99. 
Can I ask make a PR adding AC_PROG_C99 to configure.ac?